protected/views/dataset/view.php

```diff
@@ -234,9 +234,12 @@ class="btn btn-green btn-block claim-button"
                     <h5><strong><?=Yii::t('app' , 'Projects:')?></strong></h5>
                     <p>
                         <? foreach ($projects as $project){
-                            echo $project['format'];
+                            $image_url = $project['url'];
+                            $image_name = $project['name'];
+                            $image_location = preg_replace("/^http:\/\/gigadb.org/m", "", $project['image_location']);
+                            echo "<a href='$image_url' target='_blank'><img alt='Go to $image_name website' src='$image_location'/></a>";
```

Comment:
Hi @kencho51, 
making the replacement in the view template is not ideal because:

* that's more processing on the view template which will slow down the loading of that public page
* the data in the database will still be incorrect, so we will need to duplicate the fix if we want to use the same data somewhere else than on that page

I did mention database migration, but that's not the great either because this is more about content problem rather than schema, and we want to fix the data problem only once.

So it turns out that this is similar problem to the file url to replace, and adapting file-url-updater to fix project images is the better approach there.